styles.css

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
We zouden het font ook meer kunnen specificeren <h1>, <p>, enz. in plaats van op de hele body tag toepassen.